litellm/proxy/proxy_server.py

```diff
@@ -3202,7 +3203,7 @@ async def chat_completion(  # noqa: PLR0915
         data, err = modify_user_request(data)
         if err is not None:
             return err
-
+        print(f"final payload: {json.dumps(data)}")
```

Comment:
This has secrets at this point, remove this print

Reply:
removed